### PR TITLE
Fix paginator on blog page with search

### PR DESCRIPTION
**Pull request by MightyM17**
Related issue #77

### CODE REVIEW

File: templates/views/blog/_pagination.html.twig [hunk 1]
```diff
@@ -3,19 +3,19 @@
...
-					<li class="prev arrow"><a href="{{ path('blog_paginated', {page: paginator.previousPage, author: author, tag: app.request.get('tag')}) }}" rel="previous"><i class="fa fw fa-chevron-left"></i></a></li>
+    				<li class="prev arrow"><a href="{{ path('blog_paginated', {page: paginator.previousPage, author: author, tag: app.request.get('tag'), query: app.request.get('query')}) }}" rel="previous"><i class="fa fw fa-chevron-left"></i></a></li>
 				{% endif %}
 
 				{% for i in 1..paginator.lastPage %}
 					{% if i == paginator.currentPage %}
 						<li class="active"><a>{{ i }}</a></li>
 					{% else %}
-						<li><a href="{{ path('blog_paginated', {page: i, author: author, tag: app.request.get('tag')}) }}">{{ i }}</a></li>
+						<li><a href="{{ path('blog_paginated', {page: i, author: author, tag: app.request.get('tag')}), query: app.request.get('query') }}">{{ i }}</a></li>
```
Comment (Noelierx) [inline templates/views/blog/_pagination.html.twig:13]: Small typo on line 13

```suggestion
						<li><a href="{{ path('blog_paginated', {page: i, author: author, tag: app.request.get('tag'), query: app.request.get('query') }) }}">{{ i }}</a></li>
```

**Review by Noelierx — CHANGES_REQUESTED** [review]

**Comment by Noelierx** [comment]
Hello @MightyM17 thanks for your contribution! Just a small typo to fix and then I will pass the hacktoberfest accepted label :D

File: templates/views/blog/_pagination.html.twig [hunk 2]
```diff
@@ -3,19 +3,19 @@
...
 						<li class="active"><a>{{ i }}</a></li>
 					{% else %}
-						<li><a href="{{ path('blog_paginated', {page: i, author: author, tag: app.request.get('tag')}) }}">{{ i }}</a></li>
+						<li><a href="{{ path('blog_paginated', {page: i, author: author, tag: app.request.get('tag'), query: app.request.get('query') }) }}">{{ i }}</a></li>
 					{% endif %}
 				{% endfor %}
 
 				{% if paginator.hasNextPage %}
-					<li class="next arrow"><a href="{{ path('blog_paginated', {page: paginator.nextPage, author: author, tag: app.request.get('tag')}) }}" rel="next"><i class="fa fw fa-chevron-right"></i></a></li>
+					<li class="next arrow"><a href="{{ path('blog_paginated', {page: paginator.nextPage, author: author, tag: app.request.get('tag')}), query: app.request.get('query') }}" rel="next"><i class="fa fw fa-chevron-right"></i></a></li>
```
Comment (Noelierx) [inline templates/views/blog/_pagination.html.twig:18]:
```suggestion
					<li class="next arrow"><a href="{{ path('blog_paginated', {page: paginator.nextPage, author: author, tag: app.request.get('tag'), query: app.request.get('query') }) }}" rel="next"><i class="fa fw fa-chevron-right"></i></a></li>
```

**Review by Noelierx — CHANGES_REQUESTED** [review]

File: templates/views/blog/_pagination.html.twig (hunk 2 above)
Comment (Noelierx) [reply templates/views/blog/_pagination.html.twig:18]: @MightyM17 Last one, but don't worry I will commit it myself if you don't have time to do it
Also, I put the label Hacktoberfest accepted!

Comment (MightyM17) [reply templates/views/blog/_pagination.html.twig]: Fixed it, my bad for the brackets 😓

**Review by Noelierx — APPROVED** [review]

**Pull request merged**